src/modules/Plans/VMMigrationDetailsWrapper.tsx

```diff
@@ -9,7 +10,9 @@ import {
 
 const VMMigrationDetailsWrapper = withQueryClient((props: VMMigrationDetailsProps) => (
   <MustGatherContextProvider>
-    <VMMigrationDetails match={props.match} />
+    <ModalProvider>
```

Comment:
Do the `withModalProvider()` call next to the `withQueryClient()` instead to keep them all in the same HOC style.

Reply:
makes sense, fixed